clang/lib/AST/CXXInheritance.cpp

```diff
@@ -368,8 +368,8 @@ bool CXXRecordDecl::FindBaseClass(const CXXBaseSpecifier *Specifier,
                                   const CXXRecordDecl *BaseRecord) {
   assert(BaseRecord->getCanonicalDecl() == BaseRecord &&
          "User data for FindBaseClass is not canonical!");
-  return Specifier->getType()->castAs<RecordType>()->getDecl()
-            ->getCanonicalDecl() == BaseRecord;
+  return (cast<CXXRecordDecl>(Specifier->getType()->getAsRecordDecl())
+              ->getCanonicalDecl()) == BaseRecord;
```

Comment:
You do not need the outer parentheses here

Reply:
Fixed.